cypress/commons/actions/generic/ScenarioManager.js

```diff
@@ -67,10 +67,29 @@ function getScenarioEditableLabelInEditMode(scenarioId, timeout = 5) {
     timeout: timeout * 1000,
   });
 }
+
+function getScenarioEditButton(scenarioId) {
+  return getScenarioAccordion(scenarioId).find(GENERIC_SELECTORS.scenario.manager.button.edit);
```

Comment:
Could we rename `getScenarioEditButton` to `getRenameScenarioButton`?
And `GENERIC_SELECTORS.scenario.manager.button.edit` to `GENERIC_SELECTORS.scenario.manager.button.renameScenario` ?